components/UserHintBox.tsx

```diff
@@ -14,7 +14,7 @@ const UserHintBox: React.FC<Props> = ({ label, labelRight }) => {
 			>
 				<div className={`text-center h-[20%] ${styles.defaultColor}`}>
 					<div className="text-3xl">{label}</div>
-					<div className="text-xl">{labelRight}</div>
+					<div className="text-4xl">{labelRight}</div>
```

Comment:
should be 3xl since e only want a bigger font for the seconds not for the whole userHin :)